### PR TITLE
🎖️ Math magicians: full website 🎖️

### PR DESCRIPTION
**Pull request by kambereBr**
Project Requirements : https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website_v2.md

## 📝 General requirements

- ✅ No Linters error
- ✅ Used GitFlow.
- ✅ Used React Router to navigate through pages (`Home`, `Calculator` and `Quote`)
- ✅ Used styles in a React app.
- ✅ The application runs in the browser without errors

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
+import NavBar from './components/NavBar';
+import './styles/Main.css';
 
 function App() {
   return (
     <div className="App">
-      <Calculator />
-      <Quote />
+      <NavBar />
     </div>
```
Comment (Owusu-Desmond) [inline src/App.js]: - [ ] As per the project requirements, you need to create a website with three pages: Home, Calculator, and Quote. Currently, your app only has one page. Please make sure to utilize [React Router ](https://reactrouter.com/) to enable the routes to function properly. :pray: 

- [ ] Kindly note that the general layout should match these wireframes and ensure that you have a navigation bar to navigate to different routes. :blush: 
  
  image
  
  image
  
  image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @kambereBr , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Great working on the set up of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kambereBr** [comment]
@Owusu-Desmond thanks for your review 👏 
I'd like to have more details please; Do I need to have 3 more `.html` files in my project ?

**Comment by kambereBr** [comment]
@Owusu-Desmond for the usage of Route, I used it in the NavBar Component. Please, how can I enhance that?
Thanks

**Comment by Owusu-Desmond** [comment]
- [ ] Hi @kambereBr, Please I have found this article which will be helpful for you, [Adding Navigation to your React App](https://www.educative.io/blog/react-router-tutorial). The implementation of the Navigation links for routes is most likely gets done in the `App.js` component as shown below (this code snippet is here to give you an idea)
  ```jsx
  import React from 'react';
  import { BrowserRouter, Routes, Route } from 'react-router-dom';
  import Layout from '../components/Layout';
  import Home from '../components/Home';
  import Calculator from '../components/Calculator';
  import Quote from '../components/Quote';
  
  function App() {
    return (
      <div className="app">
        <BrowserRouter>
          <Routes>
            <Route path="/" element={<Layout />}>
              <Route index element={<Home />} />
              <Route path="/calculator" element={<Calculator />} />
              <Route path="/quote" element={<Quotes />} />
            </Route>
          </Routes>
        </BrowserRouter>
      </div>
    );
  }
  ```

**Comment by kambereBr** [comment]
Thanks a lot dear @Owusu-Desmond, in this project I did the same but in `NavBar.js` file like this :

  ```jsx
    import React from 'react';
    import { Routes, Route, Link } from 'react-router-dom';
    import Home from './Home';
    import Calculator from './Calculator';
    import Quote from './Quote';
    import '../styles/NavBar.css';
  
  const NavBar = () => {
    const blueColor = {
      color: 'blue',
    };
  
    return (
      <div className="container">
        <nav className="nav-bar">
          <span className="logo">Math Magicians</span>
          <ul className="nav-menu">
            <li><Link to="/" style={blueColor}>Home</Link></li>
            <li><Link to="/calculator" style={blueColor}>Calculator</Link></li>
            <li><Link to="/quote" style={blueColor}>Quote</Link></li>
          </ul>
        </nav>
        <Routes>
          <Route path="/" element={<Home />} />
          <Route path="/calculator" element={<Calculator />} />
          <Route path="/quote" element={<Quote />} />
        </Routes>
      </div>
    );
  };
  export default NavBar;
  ```

Please let me know whether I need to move this code into `App.js` file. Thanks 👏

**Comment by prolajumokeoni** [comment]
> 
Well done  so far 👏🏿

**Review by prolajumokeoni — APPROVED** [review]
Hi @kambereBr 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- Changes requested have been implemented   ✅ 
 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kambereBr** [comment]
Thanks dear @prolajumokeoni 👏

**Pull request merged**